mlrun/launcher/remote.py

```diff
@@ -162,12 +163,11 @@ def submit_job(
             logs_interval = int(
                 mlrun.mlconf.httpdb.logs.pipelines.pull_state.pull_logs_interval
             )
-
             run.wait_for_completion(
                 show_logs=True,
                 sleep=state_interval,
                 logs_interval=logs_interval,
-                raise_on_failure=False,
+                raise_on_failure=True,
```

Comment:
the thing is, if you `raise_on_failure` then you wont get to `_wrap_run_result` (in which you fixed above)

Reply:
@Tankilevitch I asked Tom to take a look when I found it, and he was not sure why he changed that in the first place, maybe that's the reason?
and we thought its best to keep the parameter even if it's the default, cause it's more clear this way.

Reply:
@yaelgen, please take a look at @liranbg suggestion, maybe that is the reason it was set as `False` in the first place, please check if when we raise_on_failure=False we will still finish the run with failure as it will get to the `_wrap_run_result` which you fixed